src/Plugin/Menu/MenuPositionLink.php

```diff
@@ -16,9 +19,24 @@ class MenuPositionLink extends MenuLinkBase {
    * @param mixed $plugin_definition
    *   The plugin implementation definition.
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityManagerInterface $entity_manager) {
     parent::__construct($configuration, $plugin_id, $plugin_definition);
+
+    $this->entity_manager = $entity_manager;
```

Comment:
For coding standards the class property should be $this->entityManager. You also need to initialize that property on the class.